amundsen_application/static/js/components/common/BadgeList/index.tsx

```diff
@@ -8,22 +8,32 @@ import { getBadgeConfig } from 'config/config-utils';
 import { Badge } from 'interfaces/Tags';
 
 export interface BadgeListProps {
-  badges: Badge[];
+  badges: any[]; // TODO replace with new badges later @allisonsuarez
 }
 
 const BadgeList: React.FC<BadgeListProps> = ({ badges }: BadgeListProps) => {
   return (
     <span className="badge-list">
       {badges.map((badge, index) => {
-        const badgeConfig = getBadgeConfig(badge.tag_name);
-
-        return (
-          <ClickableBadge
-            text={badgeConfig.displayName}
-            labelStyle={badgeConfig.style}
-            key={`badge-${index}`}
-          />
-        );
+        if (badge.tag_name) {
+          const badgeConfig = getBadgeConfig(badge.tag_name);
+          return (
+            <ClickableBadge
+              text={badgeConfig.displayName}
+              labelStyle={badgeConfig.style}
+              key={`badge-${index}`}
+            />
+          );
+        }
+        if (badge.badge_name) {
+          return (
+            <ClickableBadge
+              text={badge.badge_name}
+              labelStyle={badge.badge_type}
```

Comment:
I would convert the `badge_type` to a `BadgeStyle`. Yes, the string values do happen to be the same but our implementation shouldn't be so tightly coupled with the fact that we know exactly what metadata is sending back. `BadgeList` should have intentional logic that coverts all supported `badge_type` to a `BadgeStyle`, accounting for edge cases like mapping any unsupported or unexpected `badge_type` to the neutral style, etc.